### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by robindra-js** [community/community#58529] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I am not able to use the copilot feature and getting below error 

[INFO] [auth] [2023-06-20T08:43:52.135Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-06-20T08:43:52.138Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for using GitHub Copilot. Your subscription has ended. You are currently logged in as *****."

I checked my gitbut account and also made the payment for the subscription. Can any one help me in resolving the issue?

**Comment by KhZaza** [comment]
The error message suggests that the Copilot extension is unable to authenticate your GitHub account, even though you have an active subscription. There are a few things you can try to resolve this issue:

Make sure you are logged in to GitHub in your browser: Open your browser and navigate to github.com. Log in to your account and make sure you are logged in with the correct account that has an active Copilot subscription.

Check your Copilot token: If you're using VSCode with the Copilot extension, try checking your Copilot token by opening the command palette (Ctrl+Shift+P), selecting "GitHub Copilot: Open Settings" and then selecting "Show Copilot token". Make sure the token is valid and matches the one associated with your Copilot subscription.

Restart the Copilot extension: Try restarting the Copilot extension by closing and reopening VSCode. This can sometimes help resolve connectivity issues.

Contact GitHub support: If none of the above steps resolve the issue, you can contact GitHub support for further assistance. They should be able to help you troubleshoot the issue and ensure that your Copilot subscription is properly activated.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @robindra-js! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by davecheney** [comment, marked as answer]
@robindra-js i can see from your telemetry that you have signed in and are receiving completions. Closing

**Reply by robindra-js** [reply]
@davecheney Yes, Now I am able to connect and able to use the copilot service without any issues. 

Steps I took to fix the issue:
1. Go to Account Settings icon in the visual studio code editor.
2. Now, I signout all accounts which was showing
3. After that restart the visual studio code and this time Coopilot ask for relogin
4. Relogin with the account and it start working 

@ghostinhershell , @KhZaza Thank you for your response and suggestion on the same.
